src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue30403.cs

```diff
@@ -0,0 +1,25 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue30403 : _IssuesUITest
+{
+	public Issue30403(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "Image under WinUI does not respect VerticalOptions and HorizontalOptions with AspectFit";
+
+	[Test]
+	[Category(UITestCategories.Image)]
+	public void ImageRespectsVerticalAndHorizontalOptionsWithAspectFit()
+	{
+		App.WaitForElement("TestImage");
+
+		// Verify that the image is positioned correctly according to VerticalOptions.Center and HorizontalOptions.Center
+		// with AspectFit aspect ratio. The image should appear in the center of the Grid.
+		VerifyScreenshot();
```

Comment:
Need to include the pending reference snapshots to validate the test. Are create already in the latest build.
Example, Mac:
<img width="789" height="563" alt="image" src="https://github.com/user-attachments/assets/d0c5491f-668a-4010-a0c6-86fe2fa20525" />

Could you commit the images?